tests/src/Nightwatch/Commands/thunderSelectTag.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file
+ * Sets tag for element. It can be existing tag or new one.
```

Comment:
I think this could be more explicit about what we're dealing with. I think this is about taxonomy terms but select and tag are very very overloaded language.

Reply:
It's widget specific, so I'll move it into widget related namespace.